bin/settings/settings.env

```diff
@@ -76,10 +76,6 @@ UAA_CLIENTS_TCP_EMITTER_SECRET=uaa_client_tcp_emitter_secret
 UAA_CLIENTS_TCP_ROUTER_SECRET=uaa_client_tcp_router_secret
 USE_DIEGO_PRIVILEGED_CONTAINERS=false
 USE_STAGER_PRIVILEGED_CONTAINERS=false
-
-# HTTP_PROXY, HTTPS_PROXY, NO_PROXY are set via the transformer where applicable
 HTTP_PROXY=
-http_proxy=
 HTTPS_PROXY=
-https_proxy=
-NO_PROXY=.hcf,127.0.0.1
+NO_PROXY=.scf,127.0.0.1
```

Comment:
I don't think we need `.scf` anymore; rather, we need `.local` (because the cluster is `….cluster.local`).